fileop.go

```diff
@@ -40,6 +40,18 @@ func Delete(fileName string) error {
 	return nil
 }
 
+func FileExists(fileName string) (bool, error) {
+	var wfd wrappers.WIN32_FIND_DATA
+	handle, err := wrappers.FindFirstFile(syscall.StringToUTF16Ptr(fileName), &wfd)
```

Comment:
I would prefer GetAttributes to be used for checking for file exists. On unix systems, they use fstat usually.